sextant/Cargo.toml

```diff
@@ -15,4 +15,5 @@ northstar = {path = "../northstar"}
 npk = { path = "../npk" }
 structopt = "0.3.21"
 tempfile = "3.1.0"
+tokio = { version = "1.0.1", features = ["full"] }
```

Comment:
Here we can identify the features subset of tokio instead of using `full`. Same for the runtime